app/src/main/java/com/duckduckgo/app/usage/app/AppDaysUsedDao.kt

```diff
@@ -30,6 +30,9 @@ abstract class AppDaysUsedDao {
     @Query("SELECT COUNT(*) from app_days_used WHERE date > :isoDate")
     abstract fun getNumberOfDaysAppUsedSince(isoDate: String): Long
 
-    @Insert(onConflict = OnConflictStrategy.IGNORE)
+    @Insert(onConflict = OnConflictStrategy.REPLACE)
```

Comment:
This is changing the previous behavior, was it a bug before?

Reply:
It wasn't a bug before because we only had `date` column. However, the new approach with the second column needs REPLACE strategy as we need to update `previous_date` with the latest value.